source/adminguide/networking/advanced_zone_config.rst

```diff
@@ -49,6 +49,14 @@ configure the base guest network:
    -  **Network offering**: If the administrator has configured multiple
       network offerings, select the one you want to use for this network
 
+   - **Public MTU**: The MTU that will be configured on the public interfaces
+      of the network's VR. 
+      **NOTE:** This will not be considered for VPC network tiers, as the 
+      public MTU defined at the VPC network creation level will be considered
```

Comment:
I have a question, not related to this doc PR, but to the feature.

Have you tested public MTU in VPC VR ? @shwstppr @NuxRo

Reply:
@weizhouapache yes I tested basic deployments with multiple tiers with different MTU for each tier